server/data/feedbackClient.test.ts

```diff
@@ -0,0 +1,42 @@
+import knex from 'knex'
+import FeedbackClient from './feedbackClient'
+
+jest.mock('knex')
+
+const mockSelect = jest.fn()
+const querybuilder = {
+  select: mockSelect.mockReturnThis(),
+  where: jest.fn().mockReturnThis(),
+}
+const mockKnex = jest.fn().mockReturnValue(querybuilder)
+
+// @ts-expect-error Typescript treating variable as a type
```

Comment:
For some reason TS keeps treating this var as a Type despite the code actually working fine.